include/nccl-headers/neuron/net.h

```diff
@@ -28,10 +28,11 @@ extern "C" {
 typedef enum {
         NCCL_LOG_NONE=0,
         NCCL_LOG_VERSION=1,
-        NCCL_LOG_WARN=2,
-        NCCL_LOG_INFO=3,
-        NCCL_LOG_ABORT=4,
-        NCCL_LOG_TRACE=5
+        NCCL_LOG_FATAL=2,
+        NCCL_LOG_WARN=3,
+        NCCL_LOG_INFO=4,
+        NCCL_LOG_ABORT=5,
+        NCCL_LOG_TRACE=6
```

Comment:
This patch is too incomplete to really review.  I'm not sure I understand why you're not just using WARN for fatal errors, since that's largely how we're using it in the code base.